Library/Homebrew/cmd/audit.rb

```diff
@@ -747,6 +748,20 @@ def audit_caveats
     end
   end
 
+  def audit_reverse_migration
+    # Only enforce for new formula being re-added to core
+    return unless @strict
+    return unless formula.tap.downcase == "homebrew/homebrew"
+
+    unless TAP_MIGRATIONS.map(&:to_s).grep(/#{formula.name}/).empty?
```

Comment:
We can, but I was also planning to pull the tap information out of `TAP_MIGRATIONS` and then print that in place of the words `present tap`. Just hadn't decided the best way to do so yet :smile_cat:. PR is still a WIP, just wanted to get discussion rolling on the general idea.